.evergreen/run-orchestration.sh

```diff
@@ -27,9 +27,20 @@ ORCHESTRATION_FILE=${ORCHESTRATION_FILE}
 
 DL_START=$(date +%s)
 DIR=$(dirname $0)
-# Functions to fetch MongoDB binaries
+# Functions to fetch MongoDB binaries.
 . $DIR/download-mongodb.sh
 
+# Find python3 if we are running in bash, otherwise prefer a
+# python3 binary over python, since some modern build hosts
+# only have the python3 binary.
+export PYTHON=$(command -v python3 || command -v python)
```

Comment:
```suggestion
export PYTHON
PYTHON=$(command -v python3 || command -v python)
```
Avoid the exit code masking problem.